File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for n in range(len(nums)-1):
            for m in range(n+1,len(nums)):
                if nums[m]+nums[n]==target:
                    return( [n,m])
                    

File: 0001-two-sum/0001-two-sum.dart
class Solution {
  List<int> twoSum(List<int> nums, int target) {
    
    var seen = new Map();
    for(var i = 0; i <= nums.length; i++){
        if(seen.containsKey(target - nums[i])){
            return [i, seen[target - nums[i]]];
        }
        seen[nums[i]] = i;
    }
    return [];
    
  }
}